corehq/apps/app_manager/ui_translations/ui_translations.py

```diff
@@ -68,6 +68,10 @@ def _get_params_text(params):
         return ", ".join(["${{{}}}".format(num) for num in numbers])
 
     for row in translations:
+        if row["property"] == "":
+            warnings.append("Property '" + row["property"] + "' is empty, "
+                            "We did not add it to the translations")
```

Comment:
Should this warning message be translated? A few lines below (line 87 and below) error and warning messages are translated.

Reply:
https://github.com/dimagi/commcare-hq/pull/35969/commits/35d0bfe32e885f4a397d9afaebb74c4d1a21a0ea